src/Neo.ConsoleService/Neo.ConsoleService.csproj

```diff
@@ -1,7 +1,7 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <Version>1.2.0</Version>
+    <PackageId>Neo.ConsoleService</PackageId>
```

Comment:
this change is required?

Reply:
No, unless you want the same version to be push each time and also its doesnt append the suffix to the version. That's my reasoning. Just let me know what you want to do.